roles/wildfly_systemd/tasks/main.yml

```diff
@@ -17,6 +17,20 @@
     name: wildfly_install
     tasks_from: user.yml
 
+- name: Check if PID directory exists
+  ansible.builtin.stat:
+    path: "{{ wildfly_systemd.selinux_config.pid_path }}"
+  register: pid_directory
+
+- name: Ensure PID directory exists
```

Comment:
The name of the task is now misleading, I would suggest rename it to "Check that PID file ownership are correct". More importantly, as there is no way to know if the file in question does exist, I think we should move this into an handlers that is triggered when the service associated to wildfly move to the running state.

Reply:
@rpelisse please review it now